hyperparameters.md

```diff
@@ -32,7 +32,7 @@ ActivityCutoff: u16 = 5000;
 MaxRegistrationsPerBlock: u16 = 1;
 PruningScore : u16 = u16::MAX;
 BondsMovingAverage: u64 = 900_000;
-WeightsVersionKey: u64 = 370;
+WeightsVersionKey: u64 = 1000;
```

Comment:
Should be 510 now?

Reply:
I see 1,000 on subnet 1 @camfairchild 

<img width="1751" alt="image" src="https://github.com/opentensor/subtensor/assets/5873393/471c8a51-2e1a-4422-ad10-11e0288437d1">